README_reading_src.md

```diff
@@ -0,0 +1,542 @@
+# Cryptocurrency Tick Data Reader
+
+A comprehensive Python module for loading and analyzing cryptocurrency tick data from ZIP archives containing CSV files. This module provides easy access to historical trade tick data for various cryptocurrency pairs with pandas DataFrame support and flexible date range filtering.
+
+## 📊 Data Overview
+
+This repository contains **590+ days** of cryptocurrency trade tick data spanning from **April 2018 to November 2019**:
+
+- **7 Available Tickers**: BCCUSDT, BNBUSDT, BTCUSDT, ETHBTC, LTCBTC, NEOUSDT, QTUMUSDT
+- **Date Range**: 2018-04-07 to 2019-11-18
+- **Data Format**: Daily ZIP files containing CSV trade data
+- **Total Size**: ~5.1GB of compressed tick data
+
+### Data Structure
+
+Each trade record contains the following fields:
```

Comment:
Use the README file in root git folder as source of truth to what each header represents